app.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask
+from flask import request
+from api import openshift_info
+from flask.json import jsonify
+import signal
+import os
+import sys
+
+app = Flask(__name__)
+
+openshift_object = openshift_info.ServiceClass()
+
+@app.route('/health')
```

Comment:
Why are we keeping this? It is not checking for anything.

Reply:
As of now, i have removed this